app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java

```diff
@@ -110,8 +116,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                             != PackageManager.PERMISSION_GRANTED)) {
                 mOpenSelectImages = false; // We don't want next activity to open after getting permissions
                 requestPermissions(new String[]{Manifest.permission.WRITE_EXTERNAL_STORAGE,
-                                                Manifest.permission.READ_EXTERNAL_STORAGE,
-                                                Manifest.permission.CAMERA},
+                                Manifest.permission.READ_EXTERNAL_STORAGE,
```

Comment:
This change is causing checkstyle error. The indentation level was correct before.